src/popup.js

```diff
@@ -121,84 +122,135 @@ async function callChatGPT(question, callback, onDone) {
         onDone();
         return;
       }
-      const data = JSON.parse(message);
-      const text = data.message?.content?.parts?.[0];
-      if (text) {
-        callback(text);
+      try {
+        const data = JSON.parse(message);
+        const text = data.message?.content?.parts?.[0];
+        if (text) {
+          callback(text);
+        }
+      } catch (e) {
+        // Sometimes the API returns a datetime string, which causes the JSON parsing
+        // to fail in the middle of the output.
+        console.debug(e.message);
       }
     }
   });
 }
 const showdown = require('showdown');
 const converter = new showdown.Converter()
 
-async function reviewPR(org, repo, pr) {
+async function reviewPR(diffPath, context, title) {
   inProgress(true)
   document.getElementById('result').innerHTML = ''
-  chrome.storage.session.remove([org + repo + pr])
+  chrome.storage.session.remove([diffPath])
 
-  let patch = await fetch (`https://patch-diff.githubusercontent.com/raw/${org}/${repo}/pull/${pr}.patch`).then((r) => r.text())
+  let patch = await fetch (diffPath).then((r) => r.text())
 
   let prompt = `
-  Act as a code reviewer of a Pull Request, providing feedback on the code changes below.
-  You are provided with the Pull Request changes in a patch format.
-  Each patch entry has the commit message in the Subject line followed by the code changes (diffs) in a unidiff format.
-  \n\n
-  Patch of the Pull Request to review:
-  \n
-  ${patch}
-  \n\n
-  
+  Act as a code reviewer of a Pull Request, providing feedback on the code changes below. Do not introduce yourselves.
+  The change is has the following title: ${title}.
   As a code reviewer, your task is:
   - Review the code changes (diffs) in the patch and provide feedback.
   - If there are any bugs, highlight them.
   - Does the code do what it says in the commit messages?
   - Do not highlight minor issues and nitpicks.
-  - Use bullet points if you have multiple comments.`
+  - Use bullet points if you have multiple comments.
+  - Be as concise as possible
+  - Assume positive intent
```

Comment:
It looks like the previous bullets are all terminated with a `.`.

```suggestion
  - Be as concise as possible.
  - Assume positive intent.
```